plugin/db/mysql/mysql.go

```diff
@@ -204,6 +204,22 @@ func (driver *Driver) GetMigrationConnID(ctx context.Context) (string, error) {
 
 // Query queries a SQL statement.
 func (driver *Driver) Query(ctx context.Context, statement string, queryContext *db.QueryContext) ([]interface{}, error) {
+	singleSQLs, err := bbparser.SplitMultiSQL(bbparser.MySQL, statement)
+	if err != nil {
+		return nil, err
+	}
+	if len(singleSQLs) == 0 {
+		return nil, nil
+	}
+	// https://dev.mysql.com/doc/c-api/8.0/en/mysql-affected-rows.html
+	// If the statement is an INSERT, UPDATE, or DELETE statement, we will call execute instead of query and return the number of rows affected.
+	if len(singleSQLs) == 1 && isAffectedRowsStatement(singleSQLs[0].Text) {
+		affectedRows, err := driver.Execute(ctx, singleSQLs[0].Text, false)
+		if err != nil {
+			return nil, err
+		}
+		return []interface{}{affectedRows}, nil
```

Comment:
Can we follow the interface of sql/execute result by returning a table like result?

Reply:
We may return something like 
```json
[
    ["Affected rows"],
    ["INT"],
    [
        [ 1 ]
    ]
]
```
instead of `[ 1 ]` 
to keep the return structure consistent.